src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,47 @@
-import React from 'react';
+import cn from 'classnames';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { useLoading } from '../../hooks/useLoading';
+type Props = {
+  selectedTodo: Todo | null;
+  handleClickOfPost: () => void;
+};
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  handleClickOfPost,
+}) => {
+  const [users, setUsers] = useState<User | null>(null);
```

Comment:
The state variable `users` should be renamed to `user` to accurately reflect that it holds a single user object, not multiple users.